basedtyping/__init__.py

```diff
@@ -336,9 +362,9 @@ def issubform(
     >>> issubform(int | str, object)
     True
     """
-    # type ignores because issubclass doesn't support _SpecialForm, but we do
-    if isinstance(form, UnionType | OldUnionType):
-        for t in cast(Sequence[type], cast(UnionType, form).__args__):
+    if isinstance(form, _UnionTypes):
+        # Morally, form is an instance of "UnionType | _UnionGenericAlias"
```

Comment:
i would probably clarify that `_UnionGenericAlias` doesnt exist.

also didnt you say it can only be a union generic alias and not a `UnionType`

Reply:
it exists at runtime, but not at type time.

Reply:
I added a comment